tools/hsettp/hsettp.c

```diff
@@ -967,7 +994,13 @@ operation_usage(FILE * const output, cJSON * const path, cJSON * const method)
     operation_id = cJSON_GetStringValue(operation);
     assert(operation_id);
 
-    fprintf(output, "Usage: hsettp [OPTION ...] %s [OPTION ...]", operation_id);
+    if (x_alias) {
+        const char *alias = cJSON_GetStringValue(x_alias);
+
+        fprintf(output, "Usage: hsettp [OPTION ...] %s|%s [OPTION ...]", operation_id, alias);
```

Comment:
Not a super huge fan of this output. If anyone has better ideas, all for it.

Reply:
Does the help output show aliases?  I'm running this PR and I don't see them.  They must be shown.

Reply:
Ah.  I see them.  But only in the sub-command help.  You could show them in the main listing, something like:
```
    events       Alias for events-get
```

Reply:
I was trying to think of the best way to show them. Any thoughts?

Reply:
Maybe an `Aliases` section underneath the `Operations` section?

Reply:
Another alternative I had was a table where the first column in the operationId, second is the alias, third is the description.

Reply:
Three options listed above.  I'm fine with all of  them.

Reply:
Since Greg really cares about terminal width, I will go with the additional section I guess.